pom.xml

```diff
@@ -361,7 +361,7 @@
         <!-- Identity Local Auth BasicAuth version -->
         <identity.application.auth.basicauth.exp.pkg.version>${project.version}
         </identity.application.auth.basicauth.exp.pkg.version>
-        <identity.application.auth.basicauth.imp.pkg.version.range>[5.0.0, 6.0.0)
+        <identity.application.auth.basicauth.imp.pkg.version.range>[5.0.0, 7.0.0)
```

Comment:
shouldn't this be 6 to 7 ?

Reply:
As I have mentioned in the purpose, this property is added to define the dependency version range of the basic authenticator component for the jwt based basic authenticator component.
Due the group id update of the jwt based basic authenticator a major release was performed. But that does not include any api changes in basic authenticator component. Therefore jwt based basic authenticator can depend on basic authenticator versions ranging from [5.0.0, 7.0.0)